core/src/main/java/org/infinispan/xsite/statetransfer/XSiteStatePushCommand.java

```diff
@@ -47,11 +47,6 @@ public boolean isReturnValueExpected() {
       return false;
    }
 
-   @Override
-   public boolean canBlock() {
-      return true;
```

Comment:
I worry this `invokeAsync` may also need to use the blocking manager as the applyState method seems to be blocking from what I see.